tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -541,6 +556,24 @@ class ScopedFilterDescriptor {
   SE_DISALLOW_COPY_AND_ASSIGN(ScopedFilterDescriptor);
 };
 
+// A helper class to decide whether to enable the TENSOR_OP_MATH math type
+template <bool DefaultFlag>
+class TensorOpMath {
+ public:
+  static bool IsEnabled() {
+    static bool is_enabled = IsEnabledImpl();
+    return is_enabled;
+  }
+
+ private:
+  static bool IsEnabledImpl() {
+    using tensorflow::ReadBoolFromEnvVar;
+    bool ret;
+    ReadBoolFromEnvVar("TF_ENABLE_TENSOR_OP_MATH", DefaultFlag, &ret);
```

Comment:
Since this flag is an opt-out, maybe it would make more sense to call it TF_DISABLE_TENSOR_OP_MATH?  If I had to guess as a user, I'd expect that TF_ENABLE_TENSOR_OP_MATH=0 would be the default, but it's not...

Reply:
Done